src/rez/rezconfig.py

```diff
@@ -657,6 +657,11 @@
 # - "override": Package's preprocess function completely overrides the global preprocess.
 package_preprocess_mode = "override"
 
+# Defines whether a resolve should immediately fail if any variants have a package that can't be found.
+# It is enabled by default.
+# If disabled, it will try other variants before giving up.
+# This can be useful if you have variants that aren't available to all users.
+error_on_missing_variant_packages = True
```

Comment:
I think I would move this into the "Package Resolution" section. Also, see the comment about package vs variant.